Documentation/general/tool-nuget-package-format.md

```diff
@@ -0,0 +1,28 @@
+Tool NuGet package format
+-------------------------------------------
+
+The most straightforward way to create a .NET tool package is to run `dotnet pack` with `PackAsTool` property set in the project file. However, if your build process is highly customized, `dotnet publish` may not create the right package for you. In this case, you can create a NuGet package for your tool using a *nuspec* file and explicitly placing assets into the NuGet package following these rules.
+
+- The NuGet package has only _/tools_ folder under the root and does **not** contain any other folders; do not include folders like  _/lib_, _/content_, etc.
+- Under _/tools_ folder, the subfolders must be structured with pattern _target framework short version/RID_. For example, tool assets targeting .NET core framework V2.1 that are portable across platforms should be in the folder _tools/netcoreapp2.1/any_.
+
+Let's call assets under every _tools/target framework short version/RID_ combination per TFM-RID assets :
+- There is a DotnetToolSettings.xml for every per TFM-RID assets.
+- The package type is DotnetTool
+- Each set of TFM-RID assets should have all the dependencies the tool requires to run. The TFM-RID assets should be work correctly after being copied via `xcopy` to another machine, assuming that machine has the correct runtime version and RID environment. For portable app, there must be runtimeconfig.json for every per TFM-RID assets.
```

Comment:
Kathleen:
- Each set of TFM-RID assets should have all the dependencies the tool requires to run. The TFM-RID assets should be work correctly after being copied via `xcopy` to another machine, assuming that machine has the correct runtime version and RID environment. 

I have a question on the last sentence. Should this be a separate bullet point? Should this "must be" instead of "there is"

For portable app, there is runtimeconfig.json for every per TFM-RID assets.

--------------
Will:
Yes, "must be"

Reply:
1) Yes, last sentence separate bullet

2) grammar: should be work correctly after -> should work correctly after